node-process-argv/echo.js

```diff
@@ -0,0 +1 @@
+console.log('STOP THAT!');
```

Comment:
This should echo out what ever is included in the command to run the file. 

For example if the command: `node echo.js "Echo this out"` is ran then the output would be `Echo this out`

HINT: `process.argv` is an array, the output we want is located at index `2`